unit04/functions.rst

```diff
@@ -0,0 +1,28 @@
+Unit 4 - Functions Learning Objectives
+======================================
+
+Goals
+-----
+
+- Understand the role of functions in making code modular and reusable
+- Create programmer-defined functions
+
+Objectives
+----------
+
+Students will:
+
+- Explain what functions are and why they are important in coding.
+- Diagram how functions operate in the control flow of a program.
+- Call a function, with and without arguments.
+- Write a function with no parameters, a single parameter, and multiple
+  parameters.
+- Use descriptive names to define functions.
+- Distinguish between **parameters** and **arguments**.
+- Be able to set default values for function parameters.
+- Explain what a ``return`` statement does.
+- Use a ``return`` statement to complete the execution of a function.
+- Use a ``return`` statement to provide a value to the function call.
+- Understand *variable scope* in relation to functions.
```

Comment:
variable scope should be bold to match the other vocab terms

Reply:
Fixed, thanks.